src/relay/op/random/kernel.cc

```diff
@@ -25,21 +25,23 @@ namespace relay {
 
 TVM_REGISTER_NODE_TYPE(ThreefryGenerateAttrs);
 
+static const TensorType THREEFRY_KEY_TYPE = TensorType({10}, tvm::DataType::UInt(64));
```

Comment:
As a rule of thumb, try to avoid static variables. Use static functions that returns these variables instead. As the construction is not that costly

Reply:
Got it, is the naming style OK for a static function? `THREEFRY_KEY_TYPE()`?

Reply:
Usually CamelCase is preferred for a global or static function